src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -174,6 +174,22 @@ public class CodeStyleOptions
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("dotnet_style_prefer_is_null_check_over_reference_equality_method"),
                 new RoamingProfileStorageLocation($"TextEditor.%LANGUAGE%.Specific.{nameof(PreferIsNullCheckOverReferenceEqualityMethod)}") });
 
+        internal static readonly PerLanguageOption<CodeStyleOption<bool>> PreferConditionalExpressionOverAssignment = new PerLanguageOption<CodeStyleOption<bool>>(
+            nameof(CodeStyleOptions),
+            nameof(PreferConditionalExpressionOverAssignment),
+            defaultValue: TrueWithSuggestionEnforcement,
```

Comment:
From teh conversation here: https://github.com/dotnet/roslyn/issues/27195

In the future, we should default these to hidden/none enforcement.  It's too polarizing and offputting for people to see these suggestions show up (especially as they populate the error list).  

They're valuable to have.  And people can def opt-into higher enforcement.  But our enforcement should nto be 'suggestion' unless we're really sure that almost every customer will want this, and that the updated code is strictly better in pretty much all circumstances over the original.

@sharwell @jinujoseph @DustinCampbell  In case we want to codify this somewhere.